### PR TITLE
search page and search component

### PR DESCRIPTION
**Pull request by andjela-kostic**

### CODE REVIEW

**Review by IgorBGD92 — APPROVED** [review]

File: src/styles/search/search.scss [hunk 1]
```diff
@@ -0,0 +1,169 @@
+@import url("../variables/variables.scss");
+
+* {
+  box-sizing: border-box;
+}
+#search-head-container {
```
Comment (cerovacad) [inline src/styles/search/search.scss:6]: ps change all ids ti classes

File: src/components/Search.tsx [hunk 2]
```diff
@@ -1,7 +1,224 @@
...
+        }
+      }
+      document.addEventListener("mousedown", handleClickOutside);
+      return () => {
+        document.removeEventListener("mousedown", handleClickOutside);
+      };
+    }, [ref]);
+  }
+
+  function choseOptions(emp: Employee) {
```
Comment (cerovacad) [inline src/components/Search.tsx:85]: pls use arrow functions instead of regular ones

**Review by cerovacad — CHANGES_REQUESTED** [review]

File: src/styles/search/search.css [hunk 3]
```diff
@@ -0,0 +1,162 @@
+@import url("../variables/variables.scss");
```
Comment (cerovacad) [inline src/styles/search/search.css:1]: remove this css file pls

File: src/styles/search/search.scss [hunk 4]
```diff
@@ -3,21 +3,21 @@
 * {
```
Comment (cerovacad) [inline src/styles/search/search.scss:3]: remove this css file pls

**Review by cerovacad — APPROVED** [review]

**Pull request merged**